src/app.js

```diff
@@ -38,24 +41,140 @@ gpii.app.electronAppListener = function () {
 require("electron").app.on("ready", gpii.app.electronAppListener);
 
 
+/**
+ * Responsible for creation and housekeeping of the connection to the PCP Channel WebSocket
+ */
+fluid.defaults("gpii.app.gpiiConnector", {
+    gradeNames: "fluid.modelComponent",
+
+    // Configuration regarding the socket connection
+    config: {
+        gpiiWSUrl: "ws://localhost:8081/pcpChannel"
+    },
+
+    model: {
+        socket: "@expand:gpii.app.createGPIIConnection({that}.options.config)"
```

Comment:
This isn't a reasonable thing to put in a model since it is not serialisable - more suitable to put in "members"